bahati10/smart-home
# Setting up graphql

### What does this PR do?
setting up graphQL

### Description of Task to be completed?
#### setup graphql by adding all files relatable to the database 
#### and building a connection between the main server and graphql,
#### also added the user function to retrieve, update and delete users 

### How should this be manually tested?

1. ####  Clone the repo locally 
2. #### Start the server and make sure the DB is synced
3. #### Access this link [http://localhost:3000/graphql](http://localhost:3000/graphql) to your browser

### Any background context you want to provide?
#### N/A
### Screenshots (if appropriate) 
#### N/A
### Questions: 
#### N/A
